Archive.Zip.UnitTests/ZipFileBuilderTest.cs

```diff
@@ -280,6 +280,7 @@ public void SetFileProcessingAction_Ignore ()
       fileInfoMock.SetupGet (mock => mock.FullName).Returns (@"C:\fileName").Verifiable();
       fileInfoMock.Setup (mock => mock.Open (FileMode.Open, FileAccess.Read, FileShare.Read)).Throws (new IOException()).Verifiable();
       fileInfoMock.SetupGet (mock => mock.Parent).Returns (new DirectoryInfoWrapper (new DirectoryInfo (@"C:\")));
+      fileInfoMock.SetupGet (mock => mock.Length).Throws (new AssertionException ("Should never be called"));
```

Comment:
FYI what also works is `fileInfoMock.Verify(mock => mock.Length, Times.Never());` at the end of the test